bucket/deadbeef.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "2019-09-08",
+    "description": "A music player",
```

Comment:
There is not mentioned it is unofficial port of this player for unix based OSes

Reply:
What's your suggestion? How about `A port of DeaDBeeF music player.` ?

Reply:
> There is not mentioned it is unofficial port

http://deadbeef.sourceforge.net/download.html
> Windows builds
> DeaDBeeF builds for Windows
> Please note, that while it should be possible to build DeaDBeeF for Windows from the official git repository, the builds above are not official and not supported. Please report issues to the DeaDBeeF for Windows Project.

Reply:
So `Unofficial port of DeaDBeeF music player.` ?